app/services/queue_name_service.rb

```diff
@@ -0,0 +1,18 @@
+class QueueNameService
+  def self.call
+    new.call
+  end
+
+  def call
+    if Rails.configuration.x.environment == "uat"
+      "#{uat_queue_name}-submissions"
+    else
+      "submissions"
+    end
+  end
```

Comment:
should now be 

```
 uc-one{-possible-branch-name-onr-nil}-submissions # for all environments
```
could perhaps add (use_case [:one,  :two]) as argument to call

```
  def call(use_case)
    " uc-#{use_case}{#branch_name}-submissions"
  end
  
private
   def branch_name
    return unless if Rails.configuration.x.environment == "uat"
    Rails.configuration.x.status.app_branch.downcase.tr(" _/[]().", "-").prepend("-")
  end
```

```
QueueNameService.call(use_case)
```

Reply:
👍 Have made the change to the queue_name_service to reflect the new queue names and added use_case to call as sugggested.